src/medali/lib/plia/V01.ini

```diff
@@ -65,4 +71,7 @@ run_number: pattern, R[0-9][0-9]
 satellite_id: list, S1A, S1B
 
 # acquisition mode id
-mode: list, IW, EW
\ No newline at end of file
+mode: list, IW, EW
+
+# orbit file of which category was used
+orbit_file_type: list, POEORB, RESORB, UnknownType, None
```

Comment:
If approved, please remove "UnknownType" and None. Only POEORB or RESORB should be allowed.